src/messages/MClientRequest.h

```diff
@@ -161,6 +161,19 @@ class MClientRequest : public SafeMessage {
   int get_num_fwd() const { return head.num_fwd; }
   int get_retry_attempt() const { return head.num_retry; }
   int get_op() const { return head.op; }
+  bool is_snap_op()
```

Comment:
````
/home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/MDCache.cc: In member function 'int MDCache::path_traverse(MDRequestRef&, MDSContextFactory&, const filepath&, std::vector<CDentry*>*, CInode**, int)':
/home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/MDCache.cc:8109:28: error: passing 'const MClientRequest' as 'this' argument discards qualifiers [-fpermissive]
       if (!req->is_snap_op()) {
                            ^
In file included from /home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/MDCache.h:28,
                 from /home/jenkins-build/build/workspace/ceph-pull-requests/src/mds/MDCache.cc:23:
/home/jenkins-build/build/workspace/ceph-pull-requests/src/messages/MClientRequest.h:164:8: note:   in call to 'bool MClientRequest::is_snap_op()'
   bool is_snap_op()
        ^~~~~~~~~~
src/mds/CMakeFiles/mds.dir/build.make:192: recipe for target 'src/mds/CMakeFiles/mds.dir/MDCache.cc.o' failed
````

should be `const` method.

Reply:
By mistake, still in the stash, forgot amended it.
Thanks.